pagerduty/resource_pagerduty_event_orchestration_path_global.go

```diff
@@ -374,6 +378,7 @@ func expandGlobalPathActions(v interface{}) *pagerduty.EventOrchestrationPathRul
 		actions.Suppress = a["suppress"].(bool)
 		actions.Suspend = intTypeToIntPtr(a["suspend"].(int))
 		actions.Priority = a["priority"].(string)
+		actions.EscalationPolicy = stringTypeToStringPtr(a["escalation_policy"].(string))
```

Comment:
**Annotation:** The `EscalationPolicy` property is a string pointer (`*string`) and not a string because we need to be able to send `null` to the API in order to reset the action.